apps/storybook/src/VisCanvasInteraction.stories.tsx

```diff
@@ -39,51 +38,71 @@ const Template: Story<TemplateProps> = (args) => {
 };
 
 export const PanZoom = Template.bind({});
-PanZoom.args = { panZoom: true };
 PanZoom.argTypes = { guides: { table: { disable: true } } };
 
 export const PanZoomModifiers = Template.bind({});
 PanZoomModifiers.args = {
-  panZoom: true,
   panKey: 'Alt',
 };
-PanZoom.argTypes = { guides: { table: { disable: true } } };
+PanZoomModifiers.argTypes = { guides: { table: { disable: true } } };
+
+export const PanZoomInXAndY = Template.bind({});
+PanZoomInXAndY.args = {
+  zoomKeyX: 'Alt',
+  zoomKeyY: 'Shift',
+};
+PanZoomInXAndY.argTypes = { guides: { table: { disable: true } } };
 
 export const Tooltip = Template.bind({});
 Tooltip.args = {
+  pan: false,
+  zoom: false,
   tooltipValue: 'no value to display',
 };
 
 export const TooltipWithGuides = Template.bind({});
 TooltipWithGuides.args = {
+  pan: false,
+  zoom: false,
   tooltipValue: 'guides="both"',
   guides: 'both',
 };
 
 export const TooltipWithPanZoom = Template.bind({});
 TooltipWithPanZoom.args = {
-  panZoom: true,
```

Comment:
I could explicitely set `pan` and `zoom` to `true` rather than relying on the defaults. What do you think ?

Reply:
In the end, I made `pan` and `zoom` explicit for each story